docs/source/tutorials/index.rst

```diff
@@ -12,4 +12,5 @@ Tutorials
    audio
    dataset_health
    pred_probs_cross_val
+   knn_outlier
```

Comment:
Let's just rename it to more generic term: outliers

and the notebook to be outliers.ipynb